packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -19,17 +20,31 @@ interface ComboboxContentProps {
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
+   * */
+  readonly setSelection?: string;
```

Comment:
oh hey, it's me!

curious about the intent for this prop. 

Is it to determine which option is selected? Kinda like inputs have `value` and paired with `onChange`. In combobx terms, it's `setSelection` and `onSelection`?